src/test/java/org/gridsuite/study/server/StudyTest.java

```diff
@@ -2980,6 +2980,16 @@ public void testCreateStudyWithDefaultLoadflow() {
         assertEquals(study.getLoadFlowProvider(), defaultLoadflowProvider);
     }
 
+    @Test
+    public void testDuplicateStudy() {
+        UUID study1Uuid = createStudy("userId", CASE_UUID);
+        webTestClient.post()
+                .uri(STUDIES_URL + "?duplicateFrom={parentStudyUuid}&studyUuid={studyUuid}", study1Uuid, UUID.randomUUID())
+                .header("userId", "userId")
+                .exchange()
+                .expectStatus().isOk();
```

Comment:
Since the duplicated modifications are created alongside the study I didn't pushed any message to the broker in NetworkModificationService::createModifications, should I add a message still ?